packages/xsnap/src/xsnap.c

```diff
@@ -204,19 +204,7 @@ int main(int argc, char* argv[])
 	int error = 0;
 	int interval = 0;
 	int freeze = 0;
-	xsCreation _creation = {
-		16 * 1024 * 1024,	/* initialChunkSize */
-		16 * 1024 * 1024,	/* incrementalChunkSize */
-		1 * 1024 * 1024,	/* initialHeapCount */
-		1 * 1024 * 1024,	/* incrementalHeapCount */
-		4096,				/* stackCount */
-		32000,				/* keyCount */
-		1993,				/* nameModulo */
-		127,				/* symbolModulo */
-		8192 * 1024,		/* parserBufferSize */
-		1993,				/* parserTableModulo */
-	};
-	xsCreation* creation = &_creation;
+	int parserBufferSize = 8192 * 1024;
```

Comment:
Maybe let's make that an `unsigned int`? Since `atoi` accepts `-1234`, it might be an extra safety belt.

Reply:
you _want_ it to underflow and wrap around? I don't understand why.